docs/calendar.md

```diff
@@ -265,34 +265,36 @@ date.day         // => 7
 date.toString()  // => 2020-06-28[c=islamic]
 ```
 
-### calendar.**dateDifference**(_smaller_: Temporal.Date, _larger_: Temporal.Date, _options_: object) : Temporal.Duration
+### calendar.**dateUntil**(_one_: Temporal.Date, _two_: Temporal.Date, _options_: object) : Temporal.Duration
```

Comment:
We have both `dateAdd` and `dateSubtract` but only one `dateUntil` (not `dateSince`). Why do we have to methods in one place but only one method here?  IMHO, one method in both places might be cleanest.

Reply:
I believe we could consolidate dateAdd() and dateSubtract() once #993 is completed, but not before.

Reply:
SGTM. Filed #1066 to track this.